inc/includes.php

```diff
@@ -95,16 +95,15 @@
    // PLugin already included
    $PLUGINS_INCLUDED = 1;
    $LOADED_PLUGINS   = [];
-   $plugin           = new Plugin();
 
+   $plugin = new Plugin();
+   $plugin->init();
```

Comment:
I put back init on each page as, although it was the purpose of #5447, checks was done only on init, so it was done only on cache initialization.
Init requires to make a request to get active plugins from DB, but, most of the time, this request will be saved by the ones we gain by checking on loaded plugins array instead of DB on `Plugin::isInstalled()` and  `Plugin::isActivated()`.
For the files required to do the checks, they are the same files that are already included to load the plugin (`setup.php` for each active plugin).